README.md

```diff
@@ -170,3 +126,94 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+# Deployment to Fly.io
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, a command line interface that allows tou to do everything from creating an account to deploy to Fly.
+  - Linux:
+    ```  
+    curl  -L https://fly.io/install.sh | sh
+    ```
+  - OSX:
+    ```
+    brew install flyctl
+    ```
+  - Windows:
+    ```
+    iwr https://fly.io/install.ps1 -useb
+    ```  
+    Visit `https://fly.io/` if you need installation guide
```

Comment:
link to installation guide leads to Fly.io's homepage and not the Installation guide.